js_modules/dagster-ui/packages/ui-core/src/workspace/WorkspaceContext/WorkspaceContext.tsx

```diff
@@ -172,3 +207,21 @@ function useAllRepos(locationEntries: WorkspaceRepositoryLocationNode[]) {
     return sortBy(repos, (r) => `${r.repositoryLocation.name}:${r.repository.name}`);
   }, [locationEntries]);
 }
+
+function addAssetsData(data: PartialWorkspaceLocationNodeFragment): WorkspaceLocationNodeFragment {
+  const locationOrLoadError = data.locationOrLoadError;
+  if (locationOrLoadError?.__typename === 'RepositoryLocation') {
+    return {
+      ...data,
+      locationOrLoadError: {
+        ...locationOrLoadError,
+        repositories: locationOrLoadError.repositories.map((repo) => ({
+          ...repo,
+          assetNodes: [],
+          assetGroups: [],
+        })) as any,
```

Comment:
Can this be more specific?

Reply:
there's a weird typescript error that makes no sense to me that I couldn't figure out so I took a shortcut here :|. It makes no sense at but for some reason it thinks that the pipeline objects inside of repositories are missing some `externalPipelineSnapshotId` but it's definitely in the fragments so no idea what TS is smoking.